cmd/zc_traverser_list.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/Azure/azure-storage-azcopy/common"
+	"net/url"
+	"os"
+)
+
+// a meta traverser that goes through a list of paths (potentially directory entities) and scans them one by one
+// behaves like a single traverser (basically a "traverser of traverser")
+type listTraverser struct {
+	fileLocation            string
```

Comment:
Suggestion: make this a io.Reader or a channel so that we can use it internally. I'd like to use this to do a filepath.Glob and feed it into the listTraverser.

Reply:
Ok!